### PR TITLE
update stale documents for bnb smart chain

### PR DESCRIPTION
**Pull request by buddh0**

### CODE REVIEW

File: docs/bnb-smart-chain/developers/node_operators/archive_node.md [hunk 1]
```diff
@@ -13,10 +13,11 @@ Simply speaking, an archive node is a full node running with an additional speci
 
 Developers are limited to querying the limited recent blocks to check the balance of an address and the state of a smart contract with a full node. It is hard to get all what they want as the blockchain is moving forward at the same time, while they can query any block at a specific point in time with an archive node.
 Archive nodes are used by various applications on the blockchain for challenging use cases, including but not limited to the followings:
-* Automatic trading system needs historical data to optimize trading model
```
Comment (buddh0) [inline docs/bnb-smart-chain/developers/node_operators/archive_node.md:16]: image

not expected

File: docs/bnb-smart-chain/developers/node_operators/full_node.md [hunk 2]
```diff
@@ -64,8 +64,7 @@ We support running a full node on **Mac OS X**, **Linux**, and **Windows**.
 
     !!! note
         Make sure you use the version of geth you downloaded with wget above, and not your local installation of geth, which might be the wrong version.
-        1. Since `v1.3.1`, the flags `--txlookuplimit` has been replaced by `--history.transactions`. Make sure you no longer use `--txlookuplimit`, otherwise, node may not start.
```
Comment (buddh0) [inline docs/bnb-smart-chain/developers/node_operators/full_node.md:67]: txlookuplimit is also ok

File: docs/bnb-smart-chain/developers/quick-guide.md [hunk 3]
```diff
@@ -13,6 +13,7 @@ Since BSC is EVM-compatible, your existing Ethereum smart contract skills will s
 ## Connecting
 
 Here are some resources to help you get connected to the BSC network:
+
 - [Network Information and RPC Providers](./rpc.md)
```
Comment (buddh0) [inline docs/bnb-smart-chain/developers/quick-guide.md:17]: image
not expected

File: docs/bnb-smart-chain/introduction.md [hunk 4]
```diff
@@ -64,13 +64,6 @@ BC and BSC share the same token universe for BNB and BEP2 tokens. This defines:
 2. The total circulation of the same token should be managed across the two networks, i.e. the total effective supply of a token should be the sum of the token’s total effective supply on both BSC and BC.
 3. The tokens can be initially created on BSC in a similar format as ERC20 token standard, or on BC as a BEP2, then created on the other. There are native ways on both networks to link the two and secure the total supply of the token.
 
-## Cross-Chain Transfer and Communication
```
Comment (buddh0) [inline docs/bnb-smart-chain/introduction.md:67]: even the final sunset is not enabled
we can still not to emphasize the `Corss-Chain`

**Review by zzzckck — APPROVED** [review]

**Pull request merged**